test/CowAmmModule/CloseAmm.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.0 <0.9.0;
+
+import {CowAmmModuleTestHarness, CowAmmModule, ConstantProduct, ComposableCoW} from "./CowAmmModuleTestHarness.sol";
+import {FallbackManager} from "lib/composable-cow/lib/safe/contracts/Safe.sol";
+
+abstract contract CloseAmmTest is CowAmmModuleTestHarness {
+    function testCloseAmmWhenOneExists() public {
+        setUpDefaultCowAmm();
+
+        bytes32 previousOrderHash = cowAmmModule.activeOrders(safe);
+        require(cowAmmModule.activeOrders(safe) != bytes32(0), "no order created in setup");
```

Comment:
`singleOrders` returns false by default: just testing that a hash is zero is risky if we change some logic on how this hash is computed, since the test could pass even if the order isn't active from the start.

```suggestion
        require(cowAmmModule.activeOrders(safe) != bytes32(0), "no order created in setup");
        require(composableCow.singleOrders(address(safe), previousOrderHash), "order unset");
```